### PR TITLE
feat: auth, home timeline, like, comment, single post page

### PR DESCRIPTION
**Pull request by percius47**
basic PR for:
- auth pages
- home timeline
- follow other users
- suggested users to follow
- search box
- like, comment
- single post page

P.S: the color scheme has yet to be changed to a relatable scheme, userprofile, explore, bookmarks pages still in development
-[Live Site](https://frittr.netlify.app/)

### CODE REVIEW

File: fritter/src/features/post/components/PostCard.jsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
+import { useState, useRef } from "react";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { UserAvatar } from "../../../components/UserAvatar";
+
+import { addBookmark, removeBookmark } from "../../user/userSlice";
+import { useClickOutside } from "../../../customHooks/useClickOutside";
+// import { likedByLoggedUser, postInBookmarks, getPostDate } from "utils";
```
Comment (vijaytembugade) [inline fritter/src/features/post/components/PostCard.jsx:8]: remove comments!

File: fritter/src/features/post/components/SinglePost.jsx [hunk 2]
```diff
@@ -1,9 +1,303 @@
...
+} from "../../user/userSlice";
+import { postInBookmarks } from "../../../utils/postInBookmarks";
+import { likedByLoggedUser } from "../../../utils/likedByLoggedUser";
+import { getPostDate } from "../../../utils/getPostDate";
+import { focusInput } from "../../../utils/focusInput";
+import { CommentCard } from "./CommentCard";
+import { useClickOutside } from "../../../customHooks/useClickOutside";
+import {Sidebar} from "../../../components/Sidebar"
+import {Loader} from "../../../components/Loader"
+import { UserAvatar } from "../../../components/UserAvatar";
```
Comment (vijaytembugade) [inline fritter/src/features/post/components/SinglePost.jsx:27]: Barrel export/ import would be better here

File: fritter/src/pages/Home.jsx [hunk 3]
```diff
@@ -1,9 +1,85 @@
...
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {getPosts } from "../features/post/postSlice";
+import { getAllUsers } from "../features/user/userSlice";
+import { sortByDate } from "../utils/sortByDate";
+import { SortBar } from "../components/SortBar";
+import { Loader } from "../components/Loader";
+import { SuggestedUsers } from "../components/SuggestedUsers";
+import { Sidebar } from "../components/Sidebar";
+import { SearchBar } from "../components/SearchBar";
```
Comment (vijaytembugade) [inline fritter/src/pages/Home.jsx:10]: Here alos you can implement barrel imports

File: fritter/src/services/postServices.js [hunk 4]
```diff
@@ -55,11 +55,13 @@ const dislikePostService = ({ _id, token }) => {
 };
 
 const addCommentService = ({ postId, commentData, token }) => {
+  console.log("add comm service");
```
Comment (vijaytembugade) [inline fritter/src/services/postServices.js:58]: remove `console.log`

**Review by vijaytembugade — APPROVED** [review]
UI is clean. Good use of material UI

File: fritter/src/components/SortBar.jsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+                // setShowSortModal(false);
+              }}
+            >
+              <TrendingUpIcon/>Trending
+            </button>
+            <button
+              style={{ color: activeSort === "Latest" ? "#1d9bf0" : "" }}
+              onClick={() => {
+                dispatch(setActiveSort("Latest"));
+                // setShowSortModal(false);
```
Comment (gopalbbharadva) [inline fritter/src/components/SortBar.jsx:46]: you can remove these consoles.

File: fritter/src/features/post/components/CommentModal.jsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    (dbUser) => dbUser.username === user.username
+  );
+
+  const commentRef = useRef();
+
+  const commentId = commentExists?._id;
+
+  const addCommentHandler = (e) => {
+    e.preventDefault();
+// console.log("comment add",commentExists);
```
Comment (gopalbbharadva) [inline fritter/src/features/post/components/CommentModal.jsx:27]: remove consoles.

**Review by gopalbbharadva — APPROVED** [review]
Functionality Working fine, Well code structure.
Suggestion: add favicon and change title of your app.

**Pull request merged**